### PR TITLE
Pick a larger window size for first run.

### PR DESCRIPTION
**Pull request by vjr**
To address issue https://github.com/elementary/terminal/issues/96

### CODE REVIEW

File: src/PantheonTerminalWindow.vala [hunk 1]
```diff
@@ -325,6 +325,14 @@ namespace PantheonTerminal {
             default_width = PantheonTerminal.saved_state.window_width;
             default_height = PantheonTerminal.saved_state.window_height;
 
+            if (default_width == -1) {
+                default_width = Gdk.Screen.width () * 2 / 3;
```
Comment (cassidyjames) [inline src/PantheonTerminalWindow.vala]: Is this HiDPI-safe? I.e. if I have a 3200x1800 display with a scaling factor of 2, does `Gdk.Screen.width()` return 3200 or 1600?

Comment (cassidyjames) [inline src/PantheonTerminalWindow.vala]: I'm also curious how this behaves with multiple displays.

Comment (vjr) [inline src/PantheonTerminalWindow.vala]: Works on my hidpi display. For multiple screen maybe it needs work but existing move x y code might already have this issue anyway.

**Comment by vjr** [comment]
@cassidyjames good point about the multi screen thing. this commit https://github.com/elementary/terminal/pull/119/commits/a0c6f57f30706f2e153972436b46bbd91286e164 should address it by changing the calls of `Gdk.Screen.width ()` (which gets the default screen dimensions) to call `get_screen ().get_height ()` since this should return the screen on which the current window resides. How does that look? Note the existing code here (for centering the window) might have multi screen issue for the same code usage: https://github.com/elementary/terminal/blob/master/src/PantheonTerminalWindow.vala#L335

**Comment by cassidyjames** [comment]
@vjr that sounds better! I'll have to take a look on a HiDPI display and a few different resolutions to make sure this works well.

**Comment by vjr** [comment]
OK this branch is now "ready for review" , there's a known centering problem for multi display filed as a separate issue #121 which I'll try to work on shortly, if I can set up multi display for myself.

**Comment by vjr** [comment]
I was able to get a multi monitor setup by connecting my TV and the latest commit https://github.com/elementary/terminal/pull/119/commits/04dc0a7a1d149f429776953b5ee77d65e932553d should also address issue #121 so please test and let me know. It is working for me in my basic tests (opening first run, resizing and moving, then reopening) for hidpi, lodpi, single and dual monitor.

**Comment by vjr** [comment]
Thank you @cassidyjames . FYI I also tested on 3 computers with HiDPI, multi monitor and 1080p and 1280x800 displays and all seem to look good to me :-) Looking forward to the code review.

**Pull request merged**